migrations/2_dev_dependencies.js

```diff
@@ -1,7 +1,11 @@
 /*eslint no-undef: "off"*/
 
-const Dependencies = artifacts.require("./DevDependencies.sol")
+const migrateDependencies = require("../src/migrate_dependencies.js")
```

Comment:
What is the advantage of having the logic live somewhere else in `../`? cc @anxolin

Reply:
If someone else wants to build later on the projects, it is beneficial for them, as they can easily run the migration scripts. You could not call the migration scripts out of the migration- folder from an external project, at least to my knowledge.

Reply:
If that's the case I'm wondering if we should cross that bridge when we get to it? I find it a bit confusing to have them somewhere else. But don't feel too strongly

Reply:
Ahh, yes, I think this follows what we did in the DutchX.

The reason is:
- Truffle applies some magic (declare global objects, like `artifact`)
- To be able to allow other projects to run the same migration logic, we cannot rely on "this magic"
- The found solution is to define pure functions in the `src/migrations` dir, that do the actual migration, and have all the globals as arguments. This way, any project migrating can provide their own globals

Having said that, for the web, we won't need it probably. For the bots, I don't think so either. If there's no a better solution is a nice to have, but up too you guys if you want to include it.